tests/test_integration_queue.py

```diff
@@ -0,0 +1,108 @@
+    test_client = openreview_context["test_client"]
+
+    num_reviewers = 20
+    num_papers = 20
+    reviews_per_paper = 3
+    max_papers = 5
+    min_papers = 1
+    alternates = 0
+
+    notes_ids = []
+    for i in range(10):
+        conference_id = "AKBC.ws/{year}/Conference".format(year=3000 + i)
+        conference = clean_start_conference(
+            openreview_client,
+            conference_id,
+            num_reviewers,
+            num_papers,
+            reviews_per_paper,
+        )
+
+        reviewers_id = conference.get_reviewers_id()
+
+        config = {
+            "title": "integration-test",
+            "user_demand": str(reviews_per_paper),
+            "max_papers": str(max_papers),
+            "min_papers": str(min_papers),
+            "alternates": str(alternates),
+            "config_invitation": "{}/-/Assignment_Configuration".format(
+                reviewers_id
+            ),
+            "paper_invitation": conference.get_blind_submission_id(),
+            "assignment_invitation": conference.get_paper_assignment_id(
+                reviewers_id
+            ),
+            "deployed_assignment_invitation": conference.get_paper_assignment_id(
+                reviewers_id, deployed=True
+            ),
+            "invite_assignment_invitation": conference.get_paper_assignment_id(
+                reviewers_id, invite=True
+            ),
+            "aggregate_score_invitation": "{}/-/Aggregate_Score".format(
+                reviewers_id
+            ),
+            "conflicts_invitation": conference.get_conflict_score_id(
+                reviewers_id
+            ),
+            "custom_max_papers_invitation": "{}/-/Custom_Max_Papers".format(
+                reviewers_id
+            ),
+            "match_group": reviewers_id,
+            "scores_specification": {
+                conference.get_affinity_score_id(reviewers_id): {
+                    "weight": 1.0,
+                    "default": 0.0,
+                }
+            },
+            "status": "Initialized",
+            "solver": "FairFlow",
+        }
+
+        config_note = openreview.Note(
+            **{
+                "invitation": "{}/-/Assignment_Configuration".format(
+                    reviewers_id
+                ),
+                "readers": [conference.get_id()],
+                "writers": [conference.get_id()],
+                "signatures": [conference.get_id()],
+                "content": config,
+            }
+        )
+
+        config_note = openreview_client.post_note(config_note)
+        assert config_note
+
+        notes_ids.append(config_note.id)
+
+    for note_id in notes_ids:
+        response = test_client.post(
+            "/match",
+            data=json.dumps({"configNoteId": note_id}),
+            content_type="application/json",
+            headers=openreview_client.headers,
+        )
+        assert response.status_code == 200
+        time.sleep(1)
```

Comment:
This test currently does not check integration completely because right now tests are sent to the queue with 1 second interval which is enough time for the previous task to complete. I tried without the time interval, but the celery_worker fixture seems to be having some issues with it when using Redis as the broker. However, the test passes without the time interval if I start a live worker (not recommended).

Moreover, the test works fine without the time interval with RabbitMQ as broker.

Reply:
What is the error that you are getting from Redis? and what does it mean 'Live worker'?

You can create a larger conference with more papers and reviewers where the solver can take more time.

Reply:
With 'Live worker', I meant starting a worker using celery from another terminal. I think this might be related to the flushing of Redis. let me try again with a different Redis database as broker and backend for celery.

Reply:
So, yes, changing the redis database fixed this issue as well. I believe the problem was due to flusing of the redis by the api.